docs/UserGuide.md

```diff
@@ -244,6 +244,19 @@ Examples:
 Clientele+ saves you the hassle of sorting the list every time you open the app, so every time you do a sort, it will save the list
 </div>
 
+### Archive a Client: `archive`
```

Comment:
Would it be a good idea to remind them that to unarchive, that is a feature that is coming soon?